x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_summary.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import expect from '@kbn/expect';
+import expect from '@kbn/expect/expect';
```

Comment:
Wow, does this have an impact on the test?

Reply:
It's a new linting rule, I'm not sure what the purpose of the change is to be honest 🤔